scripts/check-size/index.js

```diff
@@ -69,12 +71,14 @@ function main() {
 
 			const newStats = bundleData;
 
-			fs.writeFileSync(
-				path.join(__dirname, '../../.parcel-ci-build.json'),
-				JSON.stringify({
-					body: processMarkdownTable(prevStats, newStats),
-				})
-			);
+			const [table, totalDiff] = generateTable(prevStats, newStats);
+
+			// eslint-disable-next-line
+			console.log(table);
+
+			if (Math.abs(totalDiff) > THRESHHOLD) {
```

Comment:
Is the rationale of the `abs` here that any big change is suspicious, including when sizes go down? (Just checking.)

Reply:
Yeah for now the idea is to fail on major changes, at least to grab the attention of the reviewer. We can hone this in more later as we evaluate how we use this. In the future I am thinking we can add per-package threshold or a user-defined limit.